app-directory/redirect-with-fallback/app/[...slug]/route.ts

```diff
@@ -0,0 +1,44 @@
+import { simpleRedirects } from '../../redirects'
+
+export const dynamic = 'error'
+export const dynamicParams = false
```

Comment:
When `dynamic` is `'error'` the page does not allow dynamic params so this line can be removed.
```suggestion
```